### PR TITLE
Edit icon swap  #3014

### PR DESCRIPTION
**Pull request by dewanhimanshu**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by dewanhimanshu** [comment]
screenshot from 2018-07-10 14-55-10
screenshot from 2018-07-10 14-55-02

**Comment by Gauravano** [comment]
I guess issue demands the heading `Link to` to be a label just like `Email` and also the position could be just before Save button. @SidharthBansal can you confirm if I am talking right? Thanks!

**Comment by SidharthBansal** [comment]
The text should be like Email.
It will be nice to have it just above save button. What do you say @jywarren?

**Comment by jywarren** [comment]
Yes, i think this sounds good. Maybe an extra line of explanation like "Link your PublicLab.org account to other providers, like Twitter, Google, Facebook or GitHub:"

**Comment by jywarren** [comment]
Looking great!!! 👍

**Comment by SidharthBansal** [comment]
@dewanhimanshu can you please update as per @jywarren 's suggestion.
Thanks

**Comment by dewanhimanshu** [comment]
ok will surely do that

**Comment by dewanhimanshu** [comment]
screenshot from 2018-07-10 21-13-30

**Comment by dewanhimanshu** [comment]
screenshot from 2018-07-10 21-19-32

**Comment by dewanhimanshu** [comment]
can you please tell me how to place icons before save. As save is being rendered by partial.

**Comment by SidharthBansal** [comment]
@jywarren I think we could have the text ``Link to`` left align instead of centre align. That will look better.
@dewanhimanshu I guess we are good with having it on the top.

**Comment by dewanhimanshu** [comment]
this is preview if we keep it left align . I haven't push it .
screenshot from 2018-07-10 22-40-31

**Comment by jywarren** [comment]
Hmm, it's odd that it's not aligned with the "Email" item, no? any idea why? 

I think left-aligning the buttons too would be great! Thanks!

**Comment by dewanhimanshu** [comment]
there was some padding . which i fixed is this ok?
screenshot from 2018-07-10 22-44-08

**Comment by SidharthBansal** [comment]
Can you please  Left align the icons and send a screenshot

**Comment by dewanhimanshu** [comment]
screenshot from 2018-07-10 23-26-34

**Comment by SidharthBansal** [comment]
@dewanhimanshu awesome.
@jywarren how will it be if we will say ``Link your Public Lab account to``. It will be short and simple. Long line appears a little weird to me. Can @dewanhimanshu @jywarren  you suggest what will be better?
Can we have another screenshot for this text? And also commit your changes.
Thanks

**Comment by SidharthBansal** [comment]
I think the icons are self-explanatory we dont need to name the providers in the heading.

**Comment by dewanhimanshu** [comment]
screenshot from 2018-07-10 23-37-58

**Comment by SidharthBansal** [comment]
Can you please add text as written above?
Also commit the changes

**Comment by dewanhimanshu** [comment]
i think we should change the color of github icon to green to black

**Comment by SidharthBansal** [comment]
You can create an issue for this. If people will agree we can change the colour. Please tag me there if you create an issue.

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -3,15 +3,19 @@
   <h2><%= t('users.edit.edit_profile') %></h2>
 
   <hr />
+  <% if Rails.env == "production"%>
+  <div class='col-lg-8 col-md-10 col-sm-10' style="display:flex; justify-content: flex-start; padding: 0;">
+  <label> Link your PublicLab.org account to other providers</label>
```
Comment (SidharthBansal) [inline app/views/users/edit.html.erb]: Can you make it short like
``Link your Public Lab account to providers``
We usually prefer short heading. Provide the last screenshot and then we are ready to go

Comment (SidharthBansal) [reply app/views/users/edit.html.erb]: Also commit your changes before you publish a screenshot.

Comment (dewanhimanshu) [reply app/views/users/edit.html.erb]: screenshot from 2018-07-11 00-03-27

Comment (dewanhimanshu) [reply app/views/users/edit.html.erb]: @SidharthBansal done

**Comment by jywarren** [comment]
This is fantastic and thanks so much for such an excellent and thorough job! We'd love your help with other issues, especially bugs!

https://github.com/publiclab/plots2/labels/bug

**Comment by SidharthBansal** [comment]
#3014

**Comment by dewanhimanshu** [comment]
@jywarren  thank you .Looking forward to solve more issues with PublicLab awesome and supporting team

**Pull request merged**